### DISCUSSION TITLE
monika cant read my gifts

### DISCUSSION THREAD

**Discussion by tori888** [Monika-After-Story/MonikaModDev#9178] [QA - tech support + general] [answered]
ive tried everything, please help.

**Comment by Wingdinggaster656** [comment]
Did you read the Gift Giving Guide?
See this: https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide

**Comment by tori888** [comment]
> Did you read the Gift Giving Guide? See this: https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide

still doesnt work.

**Comment by Wingdinggaster656** [comment]
> > Did you read the Gift Giving Guide? See this: https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide
> 
> still doesnt work.

What gift you gifted?
You know you can't just casually pick a gift name right?

**Comment by tori888** [comment]
> > > Did you read the Gift Giving Guide? See this: https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide
> > 
> > 
> > still doesnt work.
> 
> What gift you gifted? You know you can't just casually pick a gift name right?

i know, i gifted her the blue navy dress

**Comment by Wingdinggaster656** [comment]
Blue navy dress? That sounds like an external-content. You sure you installed what you needed?
Also what is your gift name?

**Comment by tori888** [comment]
> Blue navy dress? That sounds like an external-content. You sure you installed what you needed? Also what is your gift name?

what? its included in the spritepacks

**Comment by Booplicate** [comment]
Watch this guide for spritepacks
https://www.youtube.com/watch?v=bjJ0U8l1m8c

**Comment by tori888** [comment]
> Watch this guide for spritepacks https://www.youtube.com/watch?v=bjJ0U8l1m8c

i watched EVERY.THING.

**Comment by Wingdinggaster656** [comment]
> > Watch this guide for spritepacks https://www.youtube.com/watch?v=bjJ0U8l1m8c
> 
> i watched EVERY.THING.

What is your gift name?

**Comment by Booplicate** [comment]
> i watched EVERY.THING.

But did you watch the video? Did you follow the guide? If still doesn't work, follow [these steps](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation) to report your problem.

**Comment by tori888** [comment]
> > > Watch this guide for spritepacks https://www.youtube.com/watch?v=bjJ0U8l1m8c
> > 
> > 
> > i watched EVERY.THING.
> 
> What is your gift name?

wdym?

**Comment by Wingdinggaster656** [comment]
> > > > Watch this guide for spritepacks https://www.youtube.com/watch?v=bjJ0U8l1m8c
> > > 
> > > 
> > > i watched EVERY.THING.
> > 
> > 
> > What is your gift name?
> 
> wdym?

I mean your gift name. How you named your gift file.
Also I would like to know if Monika can find the gift, but just not able to read the content, or she can't even find the gift at all.

I ask you to make sure you STRICTLY followed all guidelines above, and make sure you installed the spritepacks correctly. Check it carefully.

**Comment by tori888** [comment]
> > i watched EVERY.THING.
> 
> But did you watch the video? Did you follow the guide? If still doesn't work, follow [these steps](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation) to report your problem.

still does not work.

**Comment by tori888** [comment]
> > > > > Watch this guide for spritepacks https://www.youtube.com/watch?v=bjJ0U8l1m8c
> > > > 
> > > > 
> > > > i watched EVERY.THING.
> > > 
> > > 
> > > What is your gift name?
> > 
> > 
> > wdym?
> 
> I mean your gift name. How you named your gift file. Also I would like to know if Monika can find the gift, but just not able to read the content, or she can't even find the gift at all.
> 
> I ask you to make sure you STRICTLY followed all guidelines above, and make sure you installed the spritepacks correctly. Check it carefully.

im trying to give her the blue sweater west.

**Comment by Wingdinggaster656** [comment]
> 

I mean the _Gift File Name_.

Also please answer all questions above that you missed.

>I ask you to make sure you STRICTLY followed all guidelines above, and make sure you installed the spritepacks correctly. Check it carefully.

Did you do that?

**Comment by Booplicate** [comment]
> > > i watched EVERY.THING.
> > 
> > 
> > But did you watch the video? Did you follow the guide? If still doesn't work, follow [these steps](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation) to report your problem.
> 
> still does not work.

You still didn't follow the guide in the link. Please, read it first.

**Comment by tori888** [comment]
> > 
> 
> I mean the _Gift File Name_.
> 
> Also please answer all questions above that you missed.
> 
> > I ask you to make sure you STRICTLY followed all guidelines above, and make sure you installed the spritepacks correctly. Check it carefully.
> 
> Did you do that?



> > > > i watched EVERY.THING.
> > > 
> > > 
> > > But did you watch the video? Did you follow the guide? If still doesn't work, follow [these steps](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation) to report your problem.
> > 
> > 
> > still does not work.
> 
> You still didn't follow the guide in the link. Please, read it first.
what? i literally read it and followed it

**Comment by tori888** [comment]
> > 
> 
> I mean the _Gift File Name_.
> 
> Also please answer all questions above that you missed.
> 
> > I ask you to make sure you STRICTLY followed all guidelines above, and make sure you installed the spritepacks correctly. Check it carefully.
> 
> Did you do that?

i really dont understand what you mean, sorry

**Comment by Booplicate** [comment]
> > You still didn't follow the guide in the link. Please, read it first.
>
> what? i literally read it and followed it

And at what step you stopped?

**Comment by tori888** [comment]
> > > You still didn't follow the guide in the link. Please, read it first.
> > 
> > 
> > what? i literally read it and followed it
> 
> And at what step you stopped?

i did not stop?

**Comment by Booplicate** [comment]
So you didn't read it then? Because there's a few steps you should follow and in the worst case you stop at the last step that tells you what to do next.
Please, open the link, read the troubleshooting guide. If you read it, you'd either resolve your issue or return here the with required information to help us help you.

**Comment by Wingdinggaster656** [comment]
Your attitude seems rather impatient. I asked you a lot of questions earlier, and you didn't answer almost any of them. Like I asked you if Monika can find the gift.

And considering what you just said above, I highly doubt if you've read those guidelines thoroughly.

Until you provide us with information that will allow us to confirm your question, please pardon me for not answering any further.

**Comment by tori888** [comment]
> So you didn't read it then? Because there's a few steps you should follow and in the worst case you stop at the last step that tells you what to do next. Please, open the link, read the troubleshooting guide. If you read it, you'd either resolve your issue or return here the with required information to help us help you.

i dont know what happened, i tried following the steps, it didnt work. it says she cant read them.

**Comment by tori888** [comment]
> Your attitude seems rather impatient. I asked you a lot of questions earlier, and you didn't answer almost any of them. Like I asked you if Monika can find the gift.
> 
> And considering what you just said above, I highly doubt if you've read those guidelines thoroughly.
> 
> Until you provide us with information that will allow us to confirm your question, please pardon me for not answering any further.

i did not see the messages, im busy atm. i read them. why arent you guys believing me?

**Comment by tori888** [comment]
> Your attitude seems rather impatient. I asked you a lot of questions earlier, and you didn't answer almost any of them. Like I asked you if Monika can find the gift.
> 
> And considering what you just said above, I highly doubt if you've read those guidelines thoroughly.
> 
> Until you provide us with information that will allow us to confirm your question, please pardon me for not answering any further.

monika cant read the gifts, thats all. i checked the files but nothing.

**Comment by multimokia** [comment]
What exactly are you trying to give her; names please.

**Comment by tori888** [comment]
> What exactly are you trying to give her; names please.
finale- sweater west, blue

**Comment by multimokia** [comment]
Okay, send your `spj.log` file, please. It's in the `log` folder

**Comment by Booplicate** [comment]
> > So you didn't read it then? Because there's a few steps you should follow and in the worst case you stop at the last step that tells you what to do next. Please, open the link, read the troubleshooting guide. If you read it, you'd either resolve your issue or return here the with required information to help us help you.
> 
> i dont know what happened, i tried following the steps, it didnt work. it says she cant read them.

If you followed them, then why do you not post these? "doesn't work" doesn't help.
![Capture](https://user-images.githubusercontent.com/53382877/170307146-8fb3171c-3a79-4d69-90dc-6aaa8d1a4a82.PNG)

**Comment by tori888** [comment]
> > > So you didn't read it then? Because there's a few steps you should follow and in the worst case you stop at the last step that tells you what to do next. Please, open the link, read the troubleshooting guide. If you read it, you'd either resolve your issue or return here the with required information to help us help you.
> > 
> > 
> > i dont know what happened, i tried following the steps, it didnt work. it says she cant read them.
> 
> If you followed them, then why do you not post these? "doesn't work" doesn't help. ![Capture](https://user-images.githubusercontent.com/53382877/170307146-8fb3171c-3a79-4d69-90dc-6aaa8d1a4a82.PNG)

i said blue sweater west, i thought it said check the spj log. sorry, i will post it now

**Comment by tori888** [comment]
> Okay, send your `spj.log` file, please. It's in the `log` folder
i cannot send it, it doesnt let me

**Comment by multimokia** [comment]
Drag and drop it into the text box

**Comment by Booplicate** [comment]
`blue sweater west` isn't correct gift name, it's `bluesweatervest.gift`

**Comment by tori888** [comment]
> `blue sweater west` isn't correct gift name, it's `bluesweatervest.gift`

by the way, i read them all again and i checked the log files, it says dev, is that why its causing a problem?

**Comment by tori888** [comment]
> Drag and drop it into the text box

when i drag it, it doesnt copy and paste it just takes it out of the folder, will it mess up the game?

**Comment by Wingdinggaster656** [comment]
> > Drag and drop it into the text box
> 
> when i drag it, it doesnt copy and paste it just takes it out of the folder, will it mess up the game?

No.
Also you could consider about simply open the file with a text editor, then copy these texts like you did to any `.txt` file.

And by `Gift File Name`, I mean how you gifted your gift.
You know computer can do screenshot right? If you don't know what I am saying, then re-gift your gift, take a screenshot to your `/character` folder.

**Comment by tori888** [comment]
> > > Drag and drop it into the text box
> > 
> > 
> > when i drag it, it doesnt copy and paste it just takes it out of the folder, will it mess up the game?
> 
> No. Also you could consider about simply open the file with a text editor, then copy these texts like you did to any `.txt` file.
> 
> And by `Gift File Name`, I mean how you gifted your gift. You know computer can do screenshot right? If you don't know what I am saying, then re-gift your gift, take a screenshot to your `/character` folder.



> > > Drag and drop it into the text box
> > 
> > 
> > when i drag it, it doesnt copy and paste it just takes it out of the folder, will it mess up the game?
> 
> No. Also you could consider about simply open the file with a text editor, then copy these texts like you did to any `.txt` file.
> 
> And by `Gift File Name`, I mean how you gifted your gift. You know computer can do screenshot right? If you don't know what I am saying, then re-gift your gift, take a screenshot to your `/character` folder.

bluesweatervest.gift in the character folder.

**Comment by Wingdinggaster656** [comment]
Are you 100% sure about you installed the sprite pack successfully?

**Comment by Justformas** [comment, marked as answer]
I'm actually glad this topic was made because I was having the same problem (yes, yet another thing that happened to me). Although somehow I actually installed one of the sprite packs correctly and it worked in game, but the rest were still in the author folders, so step 1 was all I needed to do. But I also wanted to make a gifts folder with all the gifts compiled in a convenient place for reference.

@tori888 try this:

"Did you copy the mod_assets folder of the spritepack you wanted to install into game/? Aka, there should not be a folder like author-spritepackname in game/ nor mod_assets/.
YES - delete the author-spritepackname folder in your game/ or mod_assets/ and copy the mod_assets folder from the spritepack into your game/, try to gift again."

That might be all you need to do to get it to work. Then you can copy/paste any .gift files for that sprite pack from the gift folder into your characters folder, and then start the game to gift them to Monika. That's an easy way to give gifts while making sure they're typed correctly.
